contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -216,6 +218,8 @@ func NewSchedulerServer() *SchedulerServer {
 		kubeletEnableDebuggingHandlers: true,
 		containPodResources:            true,
 		nodeRelistPeriod:               defaultNodeRelistPeriod,
+		conntrackTCPTimeoutEstablished: 86400, // 1 day (1/5 default), copied from cmd/kube-proxy/app/server.go
+		//conntrackMax defaults to 0 since non-zero values can trigger failures when updating /sys/module/nf_conntrack parameters (k8s#19182)
```

Comment:
I would explicitly specify 0 as the default, rather than relying the golang int default, especially if you're commenting on it.